### PR TITLE
HMOS NaturalDateFormat Library

### PR DESCRIPTION
**Pull request by vidyaakbar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
+                compile 'joda-time:joda-time:2.9.4'
+        }
+```
+
+
+## Usage
+
+#### Xml
+
+```xml
```
Comment (samuel-abhishek) [inline README.md]: xml usage code is not relevant to library. Please delete

File: README.md [hunk 2]
```diff
@@ -1,28 +1,59 @@
 # NaturalDateFormat
 
-A library to format dates as described in Material Design guidelines (https://www.google.com/design/spec/patterns/data-formats.html#data-formats-date-time).
+A HMOS library to format dates. RelativeDateFormat formats date as relative (4 months ago, today, next year), AbsoluteDateFormat formats date as absolute (Monday, 4:32 PM, June 10).
```
Comment (samuel-abhishek) [inline README.md]: "RelativeDateFormat formats date as relative (4 months ago, today, next year), AbsoluteDateFormat formats date as absolute (Monday, 4:32 PM, June 10)." can be added in the Features Section. Also explain about org.joda time lib that is used in the library.

File: naturaldateformat/src/main/java/tk/zielony/naturaldateformat/NaturalDateFormat.java [hunk 3]
```diff
@@ -1,36 +1,46 @@
...
     protected Chronology chronology;
 
-    public NaturalDateFormat(Context context, int format) {
+    protected NaturalDateFormat(Context context, int format) {
         this.context = context;
         this.format = format;
     }
 
-    public NaturalDateFormat(Context context, Chronology chronology, int format) {
+    protected NaturalDateFormat(Context context, Chronology chronology, int format) {
```
Comment (samuel-abhishek) [inline naturaldateformat/src/main/java/tk/zielony/naturaldateformat/NaturalDateFormat.java]: keep the constructors as public. Applies to all the parameterized constructors in the class.

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please check the review comments and handle. Also test and check if the strings added for different languages for internalization are functional with no issues.

File: naturaldateformat/src/main/java/tk/zielony/naturaldateformat/NaturalDateFormat.java (hunk 3 above)
Comment (vidyaakbar) [reply naturaldateformat/src/main/java/tk/zielony/naturaldateformat/NaturalDateFormat.java]: done

File: README.md (hunk 2 above)
Comment (vidyaakbar) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (vidyaakbar) [reply README.md]: done

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Code changes are ok. Test this library on sdk 6 and update the build.gradle

**Review by samuel-abhishek — APPROVED** [review]
Okay

**Review by sharathks84 — APPROVED** [review]
Okay Approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**